### PR TITLE
First draft of the Introduction to the lesson

### PR DESCRIPTION
**Pull request by mdsimpson42**
Some placeholder text. A second pair of eyes would be appreciated.

### CODE REVIEW

File: episodes/introduction.md [hunk 1]
```diff
@@ -1,114 +1,66 @@
...
-title: "Using Markdown"
+title: "Task Management for Humans"
 teaching: 10
 exercises: 2
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- How can you use tools and frameworks to work more effectively, not so you can do more work for your boss but as a form of self care?
```
Comment (olexandr-konovalov) [inline episodes/introduction.md:9]: I suggest to replace boss -> employer

File: episodes/introduction.md [hunk 2]
```diff
@@ -1,114 +1,66 @@
...
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
-
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+Developing software for research can be challenging. Many people work on multiple projects, often to tight deadlines, regularly working in fields that we know nothing about. On top of that, we have to do training, keep up with constantly evolving technologies and try to fit in extra activities to advance our careers. It will come as no surprise to most RSEs to hear that anxiety, burnout and impostor syndrome are common in this industry.
 
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+But what can we practically do to protect our mental health while working in this field? We have little control over our actual workloads, and telling a almost-burned out engineer to do some yoga or mindfulness exercises is seldom well received.
```
Comment (olexandr-konovalov) [inline episodes/introduction.md:25]: please check - do we need a hyphen in "almost-burned out"?

File: episodes/introduction.md [hunk 3]
```diff
@@ -1,114 +1,66 @@
...
-## Challenge 1: Can you do it?
+## Overview
 
-What is the output of this command?
+In the first part of this lesson, we will introduce you to a number of tools and frameworks, including:
 
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
+- Examples of
```
Comment (olexandr-konovalov) [inline episodes/introduction.md:33]: Fine for now!

File: episodes/introduction.md [hunk 4]
```diff
@@ -1,114 +1,66 @@
...
-
-## Math
-
-One of our episodes contains $\LaTeX$ equations when describing how to create
-dynamic reports with {knitr}, so we now use mathjax to describe this:
-
-`$\alpha = \dfrac{1}{(1 - \beta)^2}$` becomes: $\alpha = \dfrac{1}{(1 - \beta)^2}$
-
-Cool, right?
+This is a lesson created via The Carpentries Workbench. It is written in
```
Comment (olexandr-konovalov) [inline episodes/introduction.md:55]: Possibly not needed here, but relevant for the contributors guide?

File: episodes/introduction.md [hunk 5]
```diff
@@ -1,114 +1,66 @@
...
 
-You can use standard markdown for static figures with the following syntax:
+We are not trying to present a one-size-fits-all solution. We also no not necessarily suggest adopting these frameworks and sticking rigidly to them in their entirety. As you will see from the case studies, the best solution is often to explore these frameworks and select elements from them that work well for you. There will be things that don't work for you specifically, or which are not relevant to working in the world of research software development. 
 
-`![optional caption that appears below the figure](figure url){alt='alt text for
-accessibility purposes'}`
+## Background
 
-![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
+This lesson was initially created at [Collaborations Workshop '24](https://www.software.ac.uk/workshop/collaborations-workshop-2024-cw24), an event hosted by the [Software Sustainability Institute (SSI)](https://www.software.ac.uk/).
```
Comment (olexandr-konovalov) [inline episodes/introduction.md:51]: Possibly could be moved elsewhere to be shown at https://olexandr-konovalov.github.io/task-management/ (compare with https://carpentries.github.io/instructor-training/).

File: episodes/introduction.md (hunk 1 above)
Comment (mdsimpson42) [reply episodes/introduction.md:9]: Good point, will correct in the next check-in.

File: episodes/introduction.md (hunk 2 above)
Comment (mdsimpson42) [reply episodes/introduction.md:25]: I'm not sure. It's probably either all (almost-burned-out) or nothing (almost burned out)...

File: episodes/introduction.md (hunk 4 above)
Comment (mdsimpson42) [reply episodes/introduction.md:55]: Ok. I didn't want to delete it completely, I wasn't sure if the Carpentries would like us to include it somewhere. I will remove it for now

File: episodes/introduction.md (hunk 5 above)
Comment (mdsimpson42) [reply episodes/introduction.md:51]: Ok, where would this need to go to appear on that page?

File: episodes/introduction.md [hunk 6]
```diff
@@ -1,114 +1,66 @@
...
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
-
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+Developing software for research can be challenging. Many people work on multiple projects, often to tight deadlines, regularly working in fields that we know nothing about. On top of that, we have to do training, keep up with constantly evolving technologies and try to fit in extra activities to advance our careers. It will come as no surprise to most RSEs to hear that anxiety, burnout and impostor syndrome are common in this industry.
```
[the remark this answers is not in the document]
Comment (mdsimpson42) [reply episodes/introduction.md]: OK, I've restructured the text :thumbsup:

File: episodes/introduction.md (hunk 5 above)
Comment (olexandr-konovalov) [reply episodes/introduction.md:73]: Just to https://github.com/olexandr-konovalov/task-management/blob/main/index.md, comparing with e.g. what https://github.com/carpentries/instructor-training/blob/main/index.md does

File: index.md [hunk 7]
```diff
@@ -4,6 +4,11 @@ site: sandpaper::sandpaper_site
 
 This is a new lesson built with [The Carpentries Workbench][workbench]. 
 
+## Background
+
+This lesson was initially created at 
+[Collaborations Workshop '24](https://www.software.ac.uk/workshop/collaborations-workshop-2024-cw24), 
+an event hosted by the [Software Sustainability Institute (SSI)](https://www.software.ac.uk/).
 
 [workbench]: https://carpentries.github.io/sandpaper-docs
```
Comment (olexandr-konovalov) [inline index.md:13]: This was a reference from the text that you have deleted. I will delete it myself in the next cleanup.

**Review by olexandr-konovalov — APPROVED** [review]

**Pull request merged**